FAQ.md

```diff
@@ -0,0 +1,151 @@
+
+### Is it possible to add audio-only streams, like FM radio?
+No. Exceptions are the visual radios, in which a video and audio are shown at the same time. A WIP repository made by one member of iptv-org, [LaneSh4d0w](https://github.com/LaneSh4d0w) aims to collect radio streams, on the [IPRD](https://github.com/LaneSh4d0w/IPRD) repository.
+
+### Why are there some call sign (KJLA-DT1.us) as ids and alphanumeric names as ids (Mychannel.us)?
+It's to differentiate the origin of the broadcast from the content. Call signs usually come from physical stations in the country of origin and their programming is subject to change. If the channel is not from a television station, the brand name will be used.
+
+An example of a call sign used as id is `KJLA-DT1.us`: [KJLA](https://en.wikipedia.org/wiki/KJLA) is a digital television station, DT is a suffix, 1 is a subchannel number and ".us" is the country code (United States). This station broadcasts the Visión Latina channel, whose id is `VisionLatina.us`.
+
+### There are source for verify call sign and coverage area for some TV stations?
+There are websites to consult the location and characteristics of the station. For example, [FCCData](https://fccdata.org/) for US, Canada, Mexico, UK, Australia and Japan.
+
+### Why attributes of channels of this playlist like "tvg-country", "tvg-language", "tvg-logo" are missing?
+The reason is to reduce the workload when adding streams in the list. Since "tvg-id" links to the [Database](https://github.com/iptv-org/database) repository, each channel has unique attributes like image, name (in English and local speak), country (or countries) broadcasted, category and language.
+
+You can add multiple streams with a single id (in the "tvg-id" parameter), instead of adding information from scratch. For example:
+
+```
+#EXTINF:-1 tvg-id="CCTV3.cn" status="online",CCTV-3综艺 (1080p)
+```
+Based on [#2086](https://github.com/iptv-org/iptv/issues/2086) (countries) and [#6516](https://github.com/iptv-org/iptv/issues/6516) (use of Database).
+
+### Why some streams have an empty id?
+Because the stream does not have enough information available. If you appear to know additional data on these, let us know its official name, the language it broadcasts and the country where it originates.
+
+This also applies to the undefined.m3u file. The streams in this file are from channels whose country of origin is unknown.
+
+Based on [#2440](https://github.com/iptv-org/iptv/issues/2440).
+
+### What is a "daily update" and why are the benefits for this playlist?
+A "daily update" is a pull request made for maintenance purposes. It checks the status of the playlist operation, and reclassifies the streams internally. This will be useful on playlist recreation, as it's gonna hide streams from the playlist if they have access problems.
+
+Based on [#3057](https://github.com/iptv-org/iptv/issues/3057).
```

Comment:
```suggestion
```

The link refers to a change in the `scripts/clean.js`, it is not part of the "daily update".

Reply:
Done.